src/Avalonia.Labs.RoutedCommand/RoutedCommandBinding.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Windows.Input;
+
+namespace Avalonia.Labs.Input
+{
+    public sealed class RoutedCommandBinding : AvaloniaObject
```

Comment:
WPF CommandBinding is a bit different, where you set original ICommand and has CanExecute + Executed events. 
It can be confusing when porting code from WPF, as in this PR RoutedCommandBinding.Command is actually target command, that gets executed.

Reply:
Renamed Command to ExecutingCommand